### DISCUSSION TITLE
Lesson 6: Contract Creation Not Showing Up in MetaMask

### DISCUSSION THREAD

**Discussion by spencerj411** [Cyfrin/foundry-full-course-cu#1382] [Q&A] [answered]
I've just deployed my first contract on my local Anvil blockchain using Forge. Is It normal to not see this transaction on my MetaMask 'Activity' tab? Thanks in advance.

**Comment by Niferu** [comment, marked as answer]
Hello @spencerj411 

Yes it is normal. First of all if you are using anvil it is creating your own local blockchain, so you would need to connect to it on metamask, but even if connected contract creation is not visible in metamask even on Sepolia etc. To see all properly it is the best to check etherscan, if you are using local blockchain then just check logs and proper folders for deployments and transactions :)

**Reply by spencerj411** [reply]
Thanks for the response!
